### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by Daron976**
### In this project:
  - Created a responsive dynamic pop up window with js.
  - Dynamically inserted the work section information with js.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,78 +95,62 @@ <h3 class="ability_header">Skills</h3>
...
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
```
Comment (tufoinnkuo10) [inline index.html:132]: - [ ] Great job so far. please align your see source and see live buttons well like the figma version.
yours
Screenshot (436)_LI
figma

Screenshot (437)

File: index.html [hunk 2]
```diff
@@ -95,78 +95,62 @@ <h3 class="ability_header">Skills</h3>
...
+                    <ul class="languages projectThree">
+                        <li></li>
+                        <li></li>
+                        <li></li>
```
Comment (tufoinnkuo10) [inline index.html:136]: - [ ] please kindly make the mobile version full height like that of figma.
yours
Screenshot (439)_LI
figma
Screenshot (438)

File: script.js [hunk 3]
```diff
@@ -24,4 +24,156 @@ cancelButton.setAttribute('onclick', 'closeMenu()');
 
 for (let i = 0; i < 3; i += 1) {
   mobileLinks[i].setAttribute('onclick', 'closeMenu()');
-}
\ No newline at end of file
+}
+
+const popUpContent = [
```
Comment (tufoinnkuo10) [inline script.js:40]: - [ ] Great job with the array popup contents. Please have in mind that each array item represents a separate project you want to display. Kindly add your items according to the number of cards you have(4)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Daron976 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:132]: Implemented 👍

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:136]: Implemented 👍

File: script.js (hunk 3 above)
Comment (sja-thedude) [reply script.js:40]: - [ ] As suggested by the previous reviewer, kindly make sure that each array item represents a separate project you want to display. Kindly add your items according to the number of cards you have(4).

This means you have to make 4 arrays for your project and then display them using their arrays.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Daron976, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Daron976,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**